### PR TITLE
Loading page, no script and old browsers #79: fix

### PR DESCRIPTION
**Pull request by victorpavlenko**
https://github.com/webRunes/Default-WRIO-Theme/issues/79

### CODE REVIEW

**Comment by AlexeyAnshakov** [comment]
@michbil please review

File: WRIO-InternetOS/start.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
+var cssUrl = require('./js/global').cssUrl,
```
Comment (michbil) [inline WRIO-InternetOS/start.js]: @Xurma2 I would suggest to rename this file to preloader.js or something like that, to not confuse with start.js

File: gulpfile.js [hunk 2]
```diff
@@ -41,6 +56,9 @@ gulp.task('babel-client', ['update-modules'], function() {
         .pipe(source('start.js'))
         .pipe(gulp.dest('.'))
         .pipe(notify("start.js built!!"));
+
+
+    return merge(main, start);
```
Comment (michbil) [inline gulpfile.js]: @Xurma2  Not sure this code will work correctly with sourcemaps, merge latest changes and check

File: WRIO-InternetOS/start.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    //
+    //if(readCookie('newUser')){
+    //    document.getElementById('preloader').style.display = 'none';
+    //}else{
+    //    document.cookie = 'newUser' + '=' + 'true' + ' ' + '; path=/';
+    //}
+
+}else{
+    document.getElementById('preloader').style.display = 'none';
+}
```
Comment (michbil) [inline WRIO-InternetOS/start.js]: @Xurma2  we can try to just reqiure('main.js') if browser check succeeds

File: WRIO-InternetOS/start.js (hunk 1 above)
Comment (AlexeyAnshakov) [inline WRIO-InternetOS/start.js]: start.js is ok, because it's in html files

Comment (michbil) [inline WRIO-InternetOS/start.js]: @AlexeyAnshakov no, it's another start.js, in WRIO-InternetOS subfolder, it not exposed to html file and will be most likely confused with final start.js

Comment (AlexeyAnshakov) [inline WRIO-InternetOS/start.js]: @Xurma2 please change on preloader.js in this case, please

**Pull request merged**